ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -8,6 +8,14 @@ declaration template components/dpmlfc/schema;
 
 include { 'quattor/schema' };
 
+function component_dpmlfc_number_string_valid = {
+  if ( !match(ARGV[0],'^\d+$') ) {
+    error(format("String '%s' is not a valid number",ARGV[0]));
+    return(false);
```

Comment:
it's unneeded code and leaving it will confuse more people. i guess it's time to start to cleanup the other code too :smile: